vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -131,15 +131,16 @@ type internal FSharpAddOpenCodeFixProvider
                 openNamespaceFix context ctx name ns multipleNames)
             |> Seq.toList
             
-        let quilifySymbolFixes =
+        let qualifiedSymbolFixes =
             candidates
+            |> Seq.filter (fun (entity,_) -> not(entity.FullRelativeName.StartsWith("op_"))) // Don't include qualified operator names. The resultant codefix won't compile because it won't be an infix operator anymore.
```

Comment:
Can we somehow say StartsWith "op_" || Contains ".op_"?

I'm not at ease with idea of removing the quick fix for members containing op_ to not work.

Right now in VS2015:

![image](https://cloud.githubusercontent.com/assets/87944/23263672/386005a0-f9df-11e6-960f-4553e04fa678.png)

If there is a way to figure that out without string, that would be best but I guess for now we can have that enhanced condition.